utils/exchange.ts

```diff
@@ -0,0 +1,23 @@
+import Wei from '@synthetixio/wei';
+
+export type SynthResult = {
```

Comment:
These should probably be under types rather than utils

Reply:
Moved in the latest commit.